addons/website/views/snippets/snippets.xml

```diff
@@ -981,10 +981,6 @@
         </we-select>
     </div>
 
-    <div data-selector=".o_mega_menu .nav-link"
-         data-drop-in=".o_mega_menu nav"
-         data-drop-near=".o_mega_menu .nav-link"/>
```

Comment:
If not necessary anymore it's because the mega menu do not contain those anymore? If yes, that part should be squashed in the correct commit